rc/core/expandtab.kak

```diff
@@ -0,0 +1,38 @@
+define-command -docstring "noexpandtab: use tab character to indent and align" \
+noexpandtab %{
+    remove-hooks global noexpandtab
```

Comment:
As suggested by @Screwtapello, a single hook group would make things smaller.

Reply:
Yeah. I'll change it.